card.js

```diff
@@ -0,0 +1,37 @@
+const projects = [
+  {
+    id: 1,
+    title: 'Tonic',
+    image: 'images/portfolio-1.svg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['html', 'css', 'javaScript'],
+    source: 'https://github.com/josuerushanika/Josue-portfolio',
```

Comment:
- [ ] Great job. please you are missing the `seelive ` place holder which will serve as a link to your deployed project. the `seesource` links to your github while the `seelive` links to the deployed project.  Please add that